subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/CacheLayout.java

```diff
@@ -75,6 +75,7 @@ public enum CacheLayout {
         .changedTo(100, "8.0-milestone-5")
         .changedTo(105, "8.1-rc-2")
         .changedTo(106, "8.2-milestone-1")
+        .changedTo(107, "9.0-rc-1")
```

Comment:
📝  This is required because we no longer want to store `ForeignBuildIdentifier` type in the dependency metadata.

Alternatively, we can decide to keep the metadata version, deserialize `ForeignBuildIdentifier` fields, but recreate a `DefaultBuildIdentifier` from it.